third_party/move/move-compiler-v2/tests/checking/typing/borrow_field_from_non_struct.exp

```diff
@@ -1,49 +1,49 @@
 
 Diagnostics:
-error: type `integer` cannot be resolved as a struct
+error: constraint `<some struct>{f: ?1}` incompatible with `integer`
```

Comment:
Well. it became a bit more difficult because, as said in the PR description, field processing has now become part of type inference. However, tried to specialize this. Not perfect but should be good enough for now.